### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by Shivansh-Khunger** [Shivansh-Khunger/nitro#41] [Announcements]
## Introduction
We are excited to announce the launch of our first complete version of our NPM package. We hope you guys enjoy it 😃.
 
## What's Changed

- Support for other package mangers like `yarn` & `pnpm` was added.
- Bug where `ES-Lint` and `TS` combo created warnings was partially resolved. Check out #39 .
- A new way of showing the users which dependencies are being installed is adapted inspired by `create-next-app`.
- Successful Message is also added.
- Package Name was changed to **Nitro** as mentioned in #38 .

**Full Changelog**: https://github.com/Shivansh-Khunger/node-nitro/compare/v0.1.0-beta.7...v1.0.0

## Feedback
Your feedback is valuable to us. Please use the scripts and if you encounter any issues, please feel free to open an issue on GitHub.

## License
**nitro** is released under the MIT-Licence. See the [LICENCE ](https://github.com/Shivansh-Khunger/node-nitro?tab=MIT-1-ov-file#readme)file for more details.

<hr /><em>This discussion was created from the release <a href='https://github.com/Shivansh-Khunger/node-nitro/releases/tag/'>v1.0.0</a>.</em>

**Comment by shreverr** [comment]
yay
